design-documents/graph-ql/coverage/gift-wrapping.md

```diff
@@ -56,25 +61,40 @@ type CartPrices {
     gift_options: GiftOptionsPrices @doc(description: "The list of prices for the selected gift options")
 }
 
-type SalesTotalsInterface {
-    gift_options: GiftOptionsPrices @doc(description: "The list of prices for the selected gift options")
-}
 ###### End: Extending existing types ######
 
 
-###### Begin: Defining new types ######
-type GiftWrapping {
-    id: ID! @doc(description: "Gift wrapping unique identifier")
-    design: String! @doc(description: "Gift wrapping design name")
-    price: Money! @doc(description: "Gift wrapping price")
-    image: GiftWrappingImage! @doc(description: "Gift wrapping preview image")
+###### StoreConfig ######
+type StoreConfig {
+    allow_gift_wrapping_on_order: String @doc(description: "Allow Gift Wrapping on Order Level")
+    allow_gift_wrapping_on_order_items: String @doc(description: "Allow Gift Wrapping for Order Items")
+    allow_gift_receipt: String @doc(description: "Allow Gift Receipt")
```

Comment:
We need to replace `Configuration` section (which describes the same but in xml format) in this document with this new declaration.

Reply:
Okay, Those changes are already added to this PR magento/partners-magento2ee#246. I will change them here too.